src/Arcus.Security.Tests.Integration/KeyVault/KeyVaultSecretProviderTests.cs

```diff
@@ -0,0 +1,76 @@
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Arcus.Security.Core.Exceptions;
+using Arcus.Security.KeyVault;
+using Arcus.Security.KeyVault.Factories;
+using Arcus.Security.Tests.Integration.Logging;
+using Microsoft.Extensions.Configuration;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Arcus.Security.Tests.Integration.KeyVault
+{
+    [Trait(name: "Category", value: "Integration")]
+    public class KeyVaultSecretProviderTests
```

Comment:
Can you create the integration test build + ARM template for this as well please?
We can create an issue for it but then it will never happen/be postponed too long.

Reply:
Sorry, created an issue :) #38